### ISSUE TITLE
ImportError : cannot import name connect

### ISSUE THREAD

**Issue by Appassionato** [dronekit/dronekit-python#687]
The title says the problem. I'm trying to use dronekit on a Raspberry Pi. I have 2 of them . I tried writing a simple code to connect the vehicle but I get the error " cannot import name connect " . Then I went on my older Pi where I had a dronekit program stored there and tried it there. Only managed to run it once. Every subsequent times I get the same error. Anyone has a fix ? Thanks!

**Comment by hamishwillee** [comment]
No idea. Have you run the vehicle_state.py example in the docs?

**Comment by MKSRIRAM1993** [comment]
Hello Appassinoato... I too have same problem. when i run the scrip. it shows "cannot import name connect", but when i type the same in the python command line its working... did you fix error? if you have pls share.....

**Comment by MKSRIRAM1993** [comment]
hello hamishwillee..... 
i have installed only 2.7 version

**Comment by hamishwillee** [comment]
@MKSRIRAM1993 No, it is EXTREMELY likely you have two versions installed - probably both 2.7
Usually this bug is because the pip that you are using to install dk is different than the Pip used by the run environment. 
You can verify this by doing `python -v` in the command line and writing a script that outputs the same information. If they are different then you have two versions. 

If that isn't the problem, then I have no idea.

**Comment by MKSRIRAM1993** [comment]
thank you for the quick reply........I tried running a python script with only 
python -v
(sry i am noob to python)
i got a error " name "python" is not defined" 
so if it is different Pip, then how should i proceed...

**Comment by hamishwillee** [comment]
I updated my comment above to do it step by step. Google search is your friend.

**Comment by MKSRIRAM1993** [comment]
thank you hamishwillee....

**Comment by soufyan788** [comment]
I get this problem when i tried to run my code to connect to my drone
ImportError: cannot import name VihecleMode

can anyone help me,please
